### PR TITLE
modify target objective (db time)

### PR DESCRIPTION
**Pull request by yangdsh**
Changes:
- Add a field in MetricCatalog called `default`. It stores the metric value from the metric file in the fixture_generator. It is used for calculating the db_time with the default average wait time. This field is added to the initial migration because it is needed in the 0003 migration.
- wait_class# is Statistic
- Do not divide the metric values by observation time
- Add target objectives: RawDBTime, SummedUpDBTime, NormalizedDBTime, TransactionCounter, and ElapsedTime
- If TransactionCounter is included in the target objective, we will also normalize all the metrics by the ratio of the finished transactions and all of the transactions.

### CODE REVIEW

File: server/website/website/migrations/0001_initial.py [hunk 1]
```diff
@@ -97,6 +97,7 @@ class Migration(migrations.Migration):
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('name', models.CharField(max_length=128)),
                 ('vartype', models.IntegerField(choices=[(1, 'STRING'), (2, 'INTEGER'), (3, 'REAL'), (4, 'BOOL'), (5, 'ENUM'), (6, 'TIMESTAMP')])),
+                ('default', models.CharField(max_length=32, null=True)),
```
Comment (dvanaken) [inline server/website/website/migrations/0001_initial.py:100]: You should create a new migration file here.

File: server/website/website/db/oracle/target_objective.py [hunk 2]
```diff
@@ -4,22 +4,111 @@
...
+from website.models import DBMSCatalog, MetricCatalog
 from website.types import DBMSType
-from ..base.target_objective import BaseTargetObjective, BaseThroughput, LESS_IS_BETTER
+from ..base.target_objective import (BaseTargetObjective, BaseThroughput, LESS_IS_BETTER,
+                                     MORE_IS_BETTER)
+
+LOG = logging.getLogger(__name__)
+
+
+class SummedUpDBTime(BaseTargetObjective):
```
Comment (dvanaken) [inline server/website/website/db/oracle/target_objective.py:17]: Can you rename this? Maybe to "CustomDBTime"?

Comment (dvanaken) [reply server/website/website/db/oracle/target_objective.py:17]: Actually, it's fine with me if we rename this in a future PR. Up to you.

File: server/website/website/migrations/0001_initial.py (hunk 1 above)
Comment (yangdsh) [reply server/website/website/migrations/0001_initial.py:100]: If I create a new file after 0003, eg. 0012, 0003 reports error when it puts data into the metric_catalog from fixture files.

**Pull request merged**